lib/chef/knife/oci_server_delete.rb

```diff
@@ -82,6 +83,11 @@ def delete_chef_node(node)
         ui.msg "Deleted Chef node '#{node.name}'"
       end
 
+      def delete_chef_client(client_name)
+        object = Chef::ApiClient.load(client_name)
+        object.destroy unless object.validator
```

Comment:
What happens if the client or node with this name is not found? Is it handled in a graceful and easy to diagnose way?

Reply:
If the node isn't found, you get this message:

Instance name: [instance name]
ERROR: The object you are looking for could not be found
Response: node '[node-name]' not found

And nothing is deleted.

If the node name is found, and the client name isn't found, you get a message:

ERROR: The object you are looking for could not be found
Response: Cannot load client [client-name]

But that diagnostic message is printed after the instance is deleted, and after
the Chef node is deleted.